pulsar-client/pom.xml

```diff
@@ -182,6 +182,12 @@
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pulsar</groupId>
+      <artifactId>pulsar-broker-common</artifactId>
+      <version>2.11.0-SNAPSHOT</version>
```

Comment:
I think we should use variables here, but not a specific version `2.11.0-SNAPSHOT`

Reply:
Do you mean `${project.parent.version}`?

Reply:
OK, I see now